presidio-analyzer/app.py

```diff
@@ -0,0 +1,85 @@
+from presidio_analyzer.presidio_logger import PresidioLogger
+from presidio_analyzer.analyzer_engine import AnalyzerEngine
+from flask import Flask, request
+import json
+import os
+
+DEFAULT_PORT = "3000"
+
+WELCOME_MESSAGE = r"""
+ _______  _______  _______  _______ _________ ______  _________ _______
+(  ____ )(  ____ )(  ____ \(  ____ \\__   __/(  __  \ \__   __/(  ___  )
+| (    )|| (    )|| (    \/| (    \/   ) (   | (  \  )   ) (   | (   ) |
+| (____)|| (____)|| (__    | (_____    | |   | |   ) |   | |   | |   | |
+|  _____)|     __)|  __)   (_____  )   | |   | |   | |   | |   | |   | |
+| (      | (\ (   | (            ) |   | |   | |   ) |   | |   | |   | |
+| )      | ) \ \__| (____/\/\____) |___) (___| (__/  )___) (___| (___) |
+|/       |/   \__/(_______/\_______)\_______/(______/ \_______/(_______)
+"""
+
+
+class Server:
+
+    def __init__(self):
+        self.logger = PresidioLogger(os.environ.get("PRESIDIO_LOGGER"))
+        self.app = Flask(__name__)
+        self.logger.info("Starting analyzer engine")
+        self.engine = AnalyzerEngine()
+        self.logger.info(WELCOME_MESSAGE)
+
+        @self.app.route('/health')
+        def health() -> str:
+            """
+            Basic health probe.  get ok + 200
+            """
+            return 'ok'
+
+        @self.app.route("/analyze", methods=["POST"])
+        def analyze():
+            """
+            Executes the analyzer function
+            """
+            # Parse the request params
+            req_data = request.get_json()
+
+            text = req_data.get("text")
+            language = req_data.get("language")
+            entities = req_data.get("entities")
+            correlation_id = req_data.get("correlation_id")
+            score_threshold = req_data.get("score_threshold")
+            trace = req_data.get("trace")
```

Comment:
since all the names are the same, maybe you can use **kwargs

Reply:
I think it's a good practice not to rely on the caller, and let the app read and in some cased verify.

Reply:
Currently a missing key will be passed as None either way and you will have to validate anyway in the method.
Only difference is if the argument names won't match

Reply:
I prefer to do this middleware op, it's more safe for the long run